src/scope/real.rs

```diff
@@ -344,6 +344,7 @@ impl Scope {
     }
 
     /// Sets a tag to a specific value.
+    #[cfg_attr(feature = "cargo-clippy", allow(needless_pass_by_value))]
     pub fn set_tag<V: ToString>(&mut self, key: &str, value: V) {
```

Comment:
Didn't want to change a public-facing API, but I think this suggestion is valid (as it will always copy).